src/main/java/teammates/common/datatransfer/attributes/NotificationAttributes.java

```diff
@@ -157,12 +158,23 @@ public void setUpdatedAt(Instant updatedAt) {
     @Override
     public List<String> getInvalidityInfo() {
         // TODO: perform various checks on the fields of this object, e.g. length of title, etc.
```

Comment:
Remove this TODO as the checks are implemented.

Reply:
Was thinking of leaving this in till we finalised the length of the notification title and message body, perhaps I'll move the to-do to `FieldValidator.java`